src/clipboard_handlers/abstract_clipboard_handler.ts

```diff
@@ -12,7 +12,7 @@ import {
 export class ClipboardHandler<T> {
```

Comment:
strange behaviour with data cleanup, remove duplicate of filtered rows don't work, but from hidden rows do work. Idk if that's meant to be the case.

Reply:
Fixed :) No we have the same behavior